apps/extension/src/ui/apps/popup/pages/Portfolio/PortfolioWhatsNew.tsx

```diff
@@ -0,0 +1,111 @@
+import { HandMonoLogo } from "@talisman/theme/logos"
+import { ChevronLeftIcon } from "@talismn/icons"
+import { AnalyticsPage, sendAnalyticsEvent } from "@ui/api/analytics"
+import { useSetting } from "@ui/hooks/useSettings"
+import { useCallback } from "react"
+import { Trans, useTranslation } from "react-i18next"
+import { useNavigate } from "react-router-dom"
+
+import HeroUrl from "./assets/Whats New - Hero.png"
+
+const ANALYTICS_PAGE: AnalyticsPage = {
+  container: "Popup",
+  feature: "Portfolio",
+  featureVersion: 2,
+  page: "What's New",
+}
+
+/**
+ * This number should be incremented when we want to show the `What's New` banner
+ * to users who have dismissed it in the past.
+ *
+ * E.g. when we have new features to tell them about.
+ */
+export const WhatsNewVersion = 1
+
+export const PortfolioWhatsNew = () => {
+  const { t } = useTranslation()
+
+  return (
+    <div className="text-body-secondary flex flex-col gap-12 text-sm">
+      <div className="relative">
+        <img
+          className="pointer-events-none relative w-full rounded-sm"
+          src={HeroUrl}
+          alt="a hero banner"
+        />
+        <div className="pointer-events-none absolute left-0 top-0 flex h-full w-full flex-col gap-4 p-8">
+          <div className="font-whyteInkTrap flex items-center gap-1 text-sm tracking-tight text-white">
+            <HandMonoLogo className="text-base" />
+            Talisman
+          </div>
+          <div className="text-primary text-2xl font-extrabold">V{process.env.VERSION}</div>
```

Comment:
I don't think this should always show the latest version, but the version associated with the features listed. For example, if we release some major features in 1.X.0, it would be a bit confusing if this said 1.X.1, suggesting that the new features came out in the minor release.

To deal with this we could either code a mapping between "what's new message" and release version, or just go back to the idea of including the version number as part of the image rather than a separate html overlay.